deploy/base/config.yaml

```diff
@@ -7,3 +7,4 @@ metadata:
     security-profiles-operator/config: ""
 data:
   SPOdImagePullPolicy: Always
+  EnableSeccomp: "true"
```

Comment:
Let's add `EnableSelinux: "false"` here too. It's better to document all the available options.

Reply:
Will do.

Reply:
Done